docs/index.md

```diff
@@ -125,18 +125,64 @@ Within your existing `didFinishLaunchingWithOptions` method, add the following t
 
 Beginning with firebase-ios-sdk v9+ (react-native-firebase v15+) you must tell CocoaPods to use frameworks.
 
-Open the file `./ios/Podfile` and add this line inside your targets:
+Open the file `./ios/Podfile` and add this line inside your targets (right after `'config = use_native_modules!'`):
 
-```
-use_frameworks!
+```ruby
+use_frameworks! :linkage => :static
 ```
 
-To use Static Frameworks on iOS, you also need to manually enable this for the project with the following global to the top of your `/ios/Podfile` file:
+To use Static Frameworks on iOS, you also need to manually enable this for the project with the following global to your `/ios/Podfile` file:
 
 ```ruby
+# right after `use_frameworks! :linkage => :static`
 $RNFirebaseAsStaticFramework = true
 ```
 
+For build setting, add below codes inside your targets (right after `'__apply_Xcode_12_5_M1_post_install_workaround(installer)'`)
+
+```ruby
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+      config.build_settings["GCC_WARN_INHIBIT_ALL_WARNINGS"] = "YES"
+    end
+  end
+  
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+      config.build_settings["CC"] = "clang"
+      config.build_settings["LD"] = "clang"
+      config.build_settings["CXX"] = "clang++"
+      config.build_settings["LDPLUSPLUS"] = "clang++"
+    end
+  end
+  
+  installer.aggregate_targets.each do |aggregate_target|
+    aggregate_target.user_project.native_targets.each do |target|
+      target.build_configurations.each do |config|
+        config.build_settings['ONLY_ACTIVE_ARCH'] = 'YES'
+        config.build_settings['EXCLUDED_ARCHS'] = 'i386'
+      end
+    end
+    aggregate_target.user_project.save
+  end
```

Comment:
I'm not sure this can be a general suggestion. I think this should be in the tips-and-tricks section instead https://rnfirebase.io/faqs-and-tips

There may be cases where a developer wants to build a non-active arch, and this eliminates that capability.

Reply:
After testing, it turned out that this was not an important part... I will delete it..